init-tools.sh

```diff
@@ -164,6 +164,8 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
     echo Copying supplemental overrides from Tools-Override.
     cp $__scriptpath/Tools-Override/* $__scriptpath/Tools
 
+    Tools-Override/crossgen.sh $__scriptpath/Tools
```

Comment:
What sort of condition do you want me to use? Environment variable, script parameter, etc? Is there something uniform I should use if other scripts are doing it too?